Pipfile

```diff
@@ -26,10 +26,10 @@ django-model-utils = "*"
 djangorestframework = "*"
 whitenoise = "*"
 psycopg2-binary = "*"
-stellar-sdk = "~=2.10.0"
+stellar-sdk = "~=3.0.0"
 django-cors-headers = "*"
 toml = "*"
-pyjwt = "*"
+pyjwt = "==1.7.1"
```

Comment:
Let's upgrade it in another independent PR.

Reply:
https://github.com/stellar/django-polaris/issues/370